src/Boot/PreprocessArgs.php

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace Drush\Boot;
+
+/**
+ * Preprocess commandline arguments
+ *
+ * If we are still going to support --php and --php-options flags, then
+ * we need to remove those here as well (or add them to the Symfony
```

Comment:
I'm fine leaving behind these options for local requests. For remote requests, we might be able to ditch them as well. We have #env-vars or something like that where PATH can be manipulated.